patches/chrome-browser-extensions-install_verifier.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/extensions/install_verifier.cc b/chrome/browser/extensions/install_verifier.cc
+index 8ee620b57a0d029e0639ef8c11bc76c7ec7e888e..893dc7e3dc5999410f48ff93493b2e8629f9d69b 100644
+--- a/chrome/browser/extensions/install_verifier.cc
++++ b/chrome/browser/extensions/install_verifier.cc
+@@ -60,7 +60,7 @@ enum class VerifyStatus {
+ };
+ 
+ VerifyStatus GetExperimentStatus() {
+-#if BUILDFLAG(GOOGLE_CHROME_BRANDING) && (defined(OS_WIN) || defined(OS_MAC))
```

Comment:
An override can be used instead of a patch. For an example, see https://github.com/brave/brave-core/blob/master/chromium_src/chrome/browser/first_run/first_run_internal_posix.cc

Reply:
Fixed 😄 👍